src/sentry/static/sentry/app/utils/tokenizeSearch.tsx

```diff
@@ -299,30 +299,29 @@ function splitSearchIntoTokens(query: string) {
   const tokens: string[] = [];
 
   let token = '';
-  let endOfPrevWord = '';
-  let quoteType = '';
-  let quoteEnclosed = false;
+  let quoted = false;
 
-  queryChars.forEach((char, idx) => {
-    const nextChar = queryChars.length - 1 > idx ? queryChars[idx + 1] : null;
-    token += char;
+  for (let idx = 0; idx < queryChars.length; idx++) {
```

Comment:
@Zylphrex Do we need to update the corresponding backend code here https://github.com/getsentry/sentry/blob/ce898db90cf0a79ef3ec9991683bccdfaf8c693b/src/sentry/search/utils.py#L329-L369 ?

Reply:
afaik, they should be mirrored 1:1.

Reply:
Let me double check, you're likely right.

Reply:
I remember @EvanPurkhiser writing the corresponding frontend code in https://github.com/getsentry/sentry/pull/15812

Reply:
@wmak there actually is a comment. It's in the parent function.

Reply:
Will do, that would have helped to find this corresponding backend code.